pkg/toa/toa.go

```diff
@@ -144,3 +142,24 @@ func computeFSK(payloadSize int, frequency uint64, bitRate uint32, crc bool) (ti
 		return 0, errFrequency.WithAttributes("frequency", frequency)
 	}
 }
+
+func computeLRFHSS(phyPayloadLength int, codingRate string, crc bool) (time.Duration, error) {
```

Comment:
Could you link the spec here if there's a public link?

Reply:
We can do that, but this is defined in LoRaWAN Regional Parameters RP2 1.0.2, not in a specific datasheet like the others. We don't refer to LoRaWAN specs all the time in our codebase, so I guess that's the reason not to mention it.